### DISCUSSION TITLE
What is the best way to scrape shopping item price?

### DISCUSSION THREAD

**Discussion by hades-pro** [AAA0109/DomPicker_Extension#20] [Q&A] [answered]
I'm working on a web scraper for an e-commerce site and I need to extract the price of the shopping items.
Could you share your experience how you scrape price?

**Comment by AAA0109** [comment, marked as answer]
Often, e-commerce websites will have specific HTML tags or labels that identify price information. These tags may include “price”. These tags can be scraped using a web scraping tool, and the prices can then be extracted from the tags.

E-commerce websites often use CSS (Cascading Style Sheets) classes to style and format their prices. These classes may include "price".

Regular expressions can be used to search for patterns in the price information, such as dollar signs, commas, and decimal points. By defining specific patterns to search for, we can extract the price information more accurately.

[PROGRESS] Machine learning algorithms can be trained to identify price information by analyzing the HTML code of the page. This approach requires some programming knowledge, but it can be used to scrape prices from websites that are difficult to scrape using other methods.

**Reply by hades-pro** [reply]
Thank you.
Is it possible for scraping pricing information by not using Machine learning?
